docs/content/redis.html.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Redis
+layout: docs
+---
+
+# Redis
+
+The Flynn Redis appliance provides Redis 2.8 in a single process configuration.
+
```

Comment:
Should note that data is ephemeral, this is intended for caching and development usage.